MeetupPlatformApi/Features/Meetups/GetMeetups/Filter/FilterSettings/MeetupsFilterSettings.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace MeetupPlatformApi.Features.Meetups.GetMeetups.Filter.FilterSettings;
+
+using System.ComponentModel.DataAnnotations;
+
+public class MeetupsFilterSettings
+{
+    [Required]
+    public int PageNumber { get; set; }
+    
+    [Required]
+    public int PageSize { get; set; }
+
+    [Required]
+    public bool IsDescendingDateSort { get; set; }
```

Comment:
A very dubious decision - it would be better to use some kind of a `enum` or `static class` with `const string` fields for storing all ordering options.